tests/i18n/test_extraction.py

```diff
@@ -525,6 +528,17 @@ def test_override_plural_forms(self):
             found = re.findall(r'^(?P<value>"Plural-Forms.+?\\n")\s*$', po_contents, re.MULTILINE | re.DOTALL)
             self.assertEqual(1, len(found))
 
+    def test_trans_and_plural_blocktrans_collision(self):
+        """Ticket #17375."""
```

Comment:
Try to write a docstring so that looking up the details in the ticket isn't necessary. Include the ticket reference only if it's an obscure issue that would benefit from the additional context provided by the ticket.